viv-wallet-app/src/services/serviceBase.ts

```diff
@@ -5,11 +5,11 @@ import { getToken } from "x4b-ui";
 export class ServiceBase {
     protected http: AxiosInstance;
 
-    constructor(http?: AxiosInstance) {
+    constructor(http?: AxiosInstance, route = "/users") {
```

Comment:
you could have removed the "/" from the `route` :)
I'd even call it `entityName` ;)